src/test/java/org/xwiki/git/script/GitScriptServiceTest.java

```diff
@@ -81,37 +79,37 @@ public void setupRepository() throws Exception
     public void getRepositoryAndFindAuthors() throws Exception
     {
         GitScriptService service = this.componentManager.getInstance(ScriptService.class, "git");
-        Repository repository = service.getRepository(this.testRepository.getAbsolutePath(), TEST_REPO_CLONED,
-                "test author", "TestAccessCode");
+        Repository repository = service.getRepository(this.testRepository.getAbsolutePath(), TEST_REPO_CLONED);
         assertEquals(true, new Git(repository).pull().call().isSuccessful());
-
-        CommitFinder finder = new CommitFinder(repository);
-        CommitCountFilter count = new CommitCountFilter();
-        finder.setFilter(count);
-        finder.find();
-
-        assertEquals(1, count.getCount());
-
+        // Now find authors
         Set<PersonIdent> authors = service.findAuthors(repository);
         assertEquals(1, authors.size());
         assertEquals("test author", authors.iterator().next().getName());
     }
 
     @Test
-    public void getCountCommits() throws Exception
+    public void getRepositoryAndCountCommits() throws Exception
     {
         GitScriptService service = this.componentManager.getInstance(ScriptService.class, "git");
         Repository repository = service.getRepository(this.testRepository.getAbsolutePath(), TEST_REPO_CLONED,
-                "test author", "TestAccessCode");
+            "test author", "TestAccessCode");
         assertEquals(true, new Git(repository).pull().call().isSuccessful());
-
+        // Now count commits
         UserCommitActivity[] commits = service.countAuthorCommits(1, repository);
         // 1 author
         assertEquals(1, commits.length);
         // 1 commit
         assertEquals(1, commits[0].getCount());
-        // Verify user name and email
-        assertEquals("test author", commits[0].getName());
-        assertEquals("author@doe.com", commits[0].getEmail());
+    }
+
+    @Test
+    public void getRepositoryBareAndCheckBranch() throws Exception
```

Comment:
The naming is not quite correct. It needs to represent the fixture and not the asserts, so you should remove "AndCheckBranch".

Reply:
Got it.